### DISCUSSION TITLE
How to format/style "footer"?

### DISCUSSION THREAD

**Discussion by aborruso** [neocarto/bertin#114] [Q&A] [answered]
Hi,
I have this footer

```
    {
      type: "footer",
      text: `Lorem ipsum
2023`
    }
```

Is it possible to format it applying in example "bold" to a word, and inserting a hyperlink?

Something like that (I know, it does not work)

```
    {
      type: "footer",
      text: `Lorem **ipsum**
[2023](https://example.com)`
    }
```

Thank you

**Comment by aborruso** [comment]
@neocarto is there a way to format the footer and add also some hyperlink?

**Comment by neocarto** [comment, marked as answer]
Not yet. Could you post an issue on that topic?

**Reply by aborruso** [reply]
Sure, thank you very much!

**Reply by aborruso** [reply]
The issue https://github.com/neocarto/bertin/issues/118
